src/app/layout/header/header.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import {Component, OnInit, OnDestroy, TemplateRef} from '@angular/core';
```

Comment:
Your spacing is inconsistent with other imports - we should make sure the linter picks this up.

Reply:
it does pick it up, if and when we run it

Reply:
Nope there is no linter for the imports, but we'll add them

Reply:
There is a linter but it is off, it is on my list to turn on next.